lib/core/fs/wrapper/source_file.dart

```diff
@@ -75,16 +77,7 @@ class SourceFile {
   ///
   /// When the data is provided by a file, all line feeds are converted
   /// to `\n`.
-  Future<Uint8List> get data async {
-    if (_data != null) {
-      return _data!;
-    }
-
-    final content = await _file!.readAsString();
-    final normalized = content.replaceAll('\r\n', '\n');
-
-    return Uint8List.fromList(utf8.encode(normalized));
-  }
+  Future<Uint8List> get data async => _data ?? await _file!.readAsPosixBytes();
```

Comment:
Maybe we should convert raw data input in the constructor as well?

Reply:
We should go with the doc comment. Later on we will use the data constructor to load data we received over the network. The original data will be loaded using `SourceFile` by the front end. So the parsing has already happened